rules/manager.go

```diff
@@ -1069,7 +1074,7 @@ func (m *Manager) AlertingRules() []*AlertingRule {
 	defer m.mtx.RUnlock()
```

Comment:
Why do we need this lock at all?

Reply:
I have removed the redundant RLock in `AlertingRules` and restored that in `Rules`.